main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsStopTimeSchema.java

```diff
@@ -56,6 +56,9 @@ public interface GtfsStopTimeSchema extends GtfsEntity {
   @ForeignKey(table = "stops.txt", field = "stop_id")
```

Comment:
you should remove the @required for stopId, since stop_times.txt without stopid is unparsable. It will added back in #1749